cohort-evaluator-spark/src/test/resources/abstract-context/cql/Choice-1.0.0.cql

```diff
@@ -0,0 +1,12 @@
+library "Choice" version '1.0.0'
+using "Abstract" version '1.0.0'
+
+context AlphaNumeric
+
+define "validNumeric":
```

Comment:
Yes, that's correct.
Sorry, that's not super clear. Some of the examples ended up being more complicated because I tried to use the same `modelinfo` for both patterns.